librosa/core/convert.py

```diff
@@ -301,7 +370,7 @@ def samples_to_time(samples, *, sr=22050):
     return np.asanyarray(samples) / float(sr)
 
 
-def blocks_to_frames(blocks, *, block_length):
+def blocks_to_frames(blocks: ArrayLike, *, block_length: int) -> Union[np.integer[Any], np.ndarray]:
```

Comment:
I think this one should get an overload, as it's implicitly vectorized.  (int -> int, array -> array)

Reply:
Done